### ISSUE TITLE
Adding column names results in syntax error

### ISSUE THREAD

**Issue by asgrim** [ppomes/myanon#11]
Using SQL:

```sql
CREATE TABLE `test_with_column_names` (
    `a` int(10) unsigned NOT NULL
) ENGINE=InnoDB;
INSERT INTO `test_with_column_names` (`a`) VALUES (1);
```

And configuration:

```
# Config file for test1.sql
secret = 'lapin'
stats  = 'no'

tables = {
   `test_with_column_names` = {
     `a` = inthash 2
   }
}
```

Gives a syntax error, running with `myanon -d`, the output is:

```
main/myanon -d -f tests/test1.conf
CREATE TABLE `test_with_column_names` (
    `a` int(10) unsigned NOT NULL
) ENGINE=InnoDB;
INSERT INTO `test_with_column_names` (FOUND TABLE `test_with_column_names`

ENTERING STATE ST_TABLELOOKING FOR  `test_with_column_names`:`a`

ENTERING STATE INITIALFOUND TABLE `test_with_column_names`

ENTERING STATE ST_VALUES
Dump parsing error at line 3: syntax error - Unexpected [(]

Process finished with exit code 1
```

**Comment by ppomes** [comment]
Just to confirm, are you using the "--complete-insert" mysqldump option?

**Comment by asgrim** [comment]
> Just to confirm, are you using the "--complete-insert" mysqldump option?

Yep - `mysqldump --complete-insert --hex-blob ...`

**Comment by ppomes** [comment]
Fixed by #12
